pkg/collector/corechecks/system/iostats_nix.go

```diff
@@ -48,14 +48,14 @@ func roundFloat(val float64) float64 {
 	return math.Round(val*100) / 100
 }
 
-// Can't use values from math.MaxInt* because this should depend on the machine's word size
-const maxInt = int64(^uint(0) >> 1)
+// Can't use values from math.MaxUint* because C size for longs changes on 32/64 bit machines
+const maxLong = int64(^uint(0) >> 1)
```

Comment:
on 32-bit machines (i.e. on the machines where this'll actually be used), shouldn't this be `maxULong` instead (i.e. `math.MaxUint32`)?